estate/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Change the order to the following please, this the regular on how the manifest info is found on other modules:
```
{
    "name": "Module Name",
    "summary": """Small summary about the module""",
    "version": "1X.0.0.0.0",
    "author": "My Company",
    "website": "https://www.mycompany_web.com",
    "license": "XXXX-X",
    "depends": ["module_a", "module_b"],
    "data": [
        "views/mymodule_view.xml",
    ],
    "demo": [
        "demo/demo_data.xml",
    ],
}

```